lib/fog/libvirt/models/compute/templates/server.xml.erb

```diff
@@ -32,13 +32,36 @@
 <% end -%>
   <clock offset='utc'/>
   <devices>
+<% args = {}
+    File.readlines('/etc/foreman/ceph.conf').each do |line|
```

Comment:
What happens if the file does not exist?

Reply:
Thanks - that's handled in the commit I pushed just now.